prime-angular-frontend/src/app/shared/enums/college-licence-grouping.enum.ts

```diff
@@ -25,6 +25,36 @@ export enum CollegeLicenceGroupingEnum {
   HealthProfessionOfTraditionalChineseMedicineAcupuncture = 24
 }
 
+export const NursingLicenseGrouping: number[] = [
+  CollegeLicenceGroupingEnum.LPN,
+  CollegeLicenceGroupingEnum.RN,
+  CollegeLicenceGroupingEnum.RPN,
+  CollegeLicenceGroupingEnum.NP,
+  CollegeLicenceGroupingEnum.MW,
+]
+
+export const NonNursingLicenseGrouping: number[] = [
+  CollegeLicenceGroupingEnum.CertifiedDentalAssistant,
+  CollegeLicenceGroupingEnum.DentalHygienist,
+  CollegeLicenceGroupingEnum.DentalTechnician,
+  CollegeLicenceGroupingEnum.DentalTherapist,
+  CollegeLicenceGroupingEnum.Dentist,
+  CollegeLicenceGroupingEnum.Denturist,
+  CollegeLicenceGroupingEnum.HealthProfessionOfDietetics,
+  CollegeLicenceGroupingEnum.HealthProfessionOfOccupationalTherapy,
+  CollegeLicenceGroupingEnum.HealthProfessionOfOpticianry,
+  CollegeLicenceGroupingEnum.HealthProfessionOfOptomety,
+  CollegeLicenceGroupingEnum.HealthProfessionOfPhysicalTherapy,
+  CollegeLicenceGroupingEnum.HealthProfessionOfPsychology,
+  CollegeLicenceGroupingEnum.HealthProfessionOfAudiology,
+  CollegeLicenceGroupingEnum.HealthProfessionOfHearingInstrumentDispensing,
+  CollegeLicenceGroupingEnum.HealthProfessionOfSpeechLanguagePathology,
+  CollegeLicenceGroupingEnum.HealthProfessionOfChiropractic,
+  CollegeLicenceGroupingEnum.HealthProfessionOfMassageTherapy,
+  CollegeLicenceGroupingEnum.HealthProfessionOfNaturopathicMedicine,
+  CollegeLicenceGroupingEnum.HealthProfessionOfTraditionalChineseMedicineAcupuncture
+]
+
 export const LicenseGrouping: number[] = [
```

Comment:
Can `LicenseGrouping` be a union of `NursingLicenseGrouping` and `NonNursingLicenseGrouping` instead?

Reply:
Sure. Code is updated.